src/commands/deployWorkspaceProject/deployWorkspaceProject.ts

```diff
@@ -67,13 +67,13 @@ export async function deployWorkspaceProject(context: IActionContext): Promise<v
     if (wizardContext.resourceGroup) {
         const resourceGroupName: string = nonNullValueAndProp(wizardContext.resourceGroup, 'name');
 
-        // wizardContext.activityChildren?.push(
-        //     new GenericTreeItem(undefined, {
-        //         contextValue: createActivityChildContext(['useExistingResourceGroup', activitySuccessContext]),
-        //         label: localize('useResourceGroup', 'Use resource group "{0}"', resourceGroupName),
-        //         iconPath: activitySuccessIcon
-        //     })
-        // );
+        wizardContext.activityChildren?.push(
+            new GenericTreeItem(undefined, {
+                contextValue: createActivityChildContext(['useExistingResourceGroup', activitySuccessContext]),
+                label: localize('useResourceGroup', 'Use resource group "{0}"', resourceGroupName),
+                iconPath: activitySuccessIcon
```

Comment:
This is bit of an aside, but should we have another icon for children that are just relaying information? It's kind of weird to me that we'd have green checkmarks for all of these. 

Not something we need to decide on now, just curious.

Reply:
Yeah, I agree, it might look cool to experiment with the icons for these

Reply:
Just playing around with the icon... and these are out of order, but thoughts?

<img width="818" alt="image" src="https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/73b93573-6665-4e42-8fef-5d10a1ba9671">

Reply:
Yeah, I do prefer this. It's still a bit busy in my opinion, but better.

Maybe even just a dash or a bullet point would work?